scripts/bootstrap.sh

```diff
@@ -364,6 +364,104 @@ create_user_schema_property() {
   fi
 }
 
+update_group_attributes() {
+  local group_id="$1"
+  local attributes_json="$2"
+  
+  # shellcheck disable=SC2016
+  local query='{"query":"mutation UpdateGroup($group: UpdateGroupInput!) {updateGroup(group: $group) {ok}}","operationName":"UpdateGroup"}'
```

Comment:
Are you using this variable? It seems repeated below

Reply:
Good catch. I made a change to make the construction of the query more readable using jq.